tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -106,41 +105,77 @@ func TestWorkloadEntryUpdatesStatus(t *testing.T) {
 				Revision: "",
 				Labels:   nil,
 			})
-			// Make a workloadentry
-			ctx.Config().ApplyYAMLOrFail(t, ns.Name(), `
+
+			// create WorkloadEntry
+		ctx.Config().ApplyYAMLOrFail(t, ns.Name(), `
 apiVersion: networking.istio.io/v1alpha3
 kind: WorkloadEntry
 metadata:
   name: vm-1
+spec:
+  address: 127.0.0.1
 `)
-			// Test adding one condition
-			stat := &v1alpha1.IstioStatus{
-				Conditions: []*v1alpha1.IstioCondition{
+
+			retry.UntilSuccessOrFail(t, func() error {
+				// we should expect an empty array not nil
+				return expectWorkloadEntryStatus(t, ctx, ns, nil)
```

Comment:
We should expect that condition Reconciled and condition PassedAnalysis per https://istio.io/latest/docs/reference/config/config-status/.  Is that not what you are seeing?

Reply:
no, im seeing a nil condition array for some reason. Is this an issue with my test?

Do I have to call something to trigger those conditions?